src/fu-plugin.c

```diff
@@ -100,7 +100,12 @@ typedef gboolean	 (*FuPluginUsbDeviceAddedFunc)	(FuPlugin	*plugin,
 typedef gboolean	 (*FuPluginUdevDeviceAddedFunc)	(FuPlugin	*plugin,
 							 GUdevDevice	*udev_device,
 							 GError		**error);
-
+typedef gboolean	 (*FuPluginUsbDeviceRemovedFunc)(FuPlugin	*plugin,
```

Comment:
Isn't this just the same as FuPluginUsbDeviceAddedFunc? Can't we just rename that to `FuPluginUdevDeviceFunc`?

Reply:
fixed